devel/moth.py

```diff
@@ -59,6 +59,34 @@ def __init__(self, stream, name, visible=True):
         self.name = name
         self.visible = visible
 
+class PuzzleSuccess(dict):
```

Comment:
Why not use a namedTuple?

Reply:
namedtuple doesn't cleanly convert to a form that can be JSON-serialized, but I still wanted a way to enforce a schema